tls/s2n_ktls.c

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include "tls/s2n_ktls.h"
+
+bool platform_supports_ktls()
```

Comment:
This needs the "s2n" prefix. I'd suggest:
```suggestion

bool s2n_ktls_is_supported()
```
Your other "ktls is supported" checks will probably be at the connection or config level, right? Like s2n_connection_is_ktls_supported?

Reply:
I kept `platform` since ktls_supported is a complex question and depends on cipher, tls version, platform, pre/post handshake and I want to make it clear that this is asking about platform.